lightgbm/src/test/scala/com/microsoft/azure/synapse/ml/lightgbm/split2/VerifyLightGBMRankerBulk.scala

```diff
@@ -0,0 +1,13 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.azure.synapse.ml.lightgbm.split2
+
+import com.microsoft.azure.synapse.ml.lightgbm.LightGBMConstants
+
+/** Tests to validate the functionality of LightGBM Ranker module in bulk mode. */
+class VerifyLightGBMRankerBulk extends VerifyLightGBMRankerStream {
+  override lazy val executionMode: String = LightGBMConstants.BulkExecutionMode
+  override def ignoreSerializationFuzzing: Boolean = true
+  override def ignoreExperimentFuzzing: Boolean = true
+}
```

Comment:
This looks a little odd, if you are sharing an implementation and just changing this execution mode, perhaps make the base class an actual base class instead of having bulk inherit stream. It will add 1 extra class, but it should be more in-line with the actual abstraction

Reply:
fair enough, I think I did it this way in anticipation that one day we'd just remove bulk. :)

Oh I know why I did it this way... if you make an abstract base class, you can't click on individual tests and run them in IJ. It was for convenience of being able to run individual tests.